src/classes/ADDR_Addresses_TDTM.cls

```diff
@@ -366,31 +366,44 @@ public class ADDR_Addresses_TDTM extends TDTM_Runnable {
     /*******************************************************************************************************
     * @description utility to return the current StartDate or EndDate of a Seasonal Address
     * @param addr the Address to look at
-    * @param fStart whether to return the StartDate or EndDate
+    * @param returnStartDate whether to return the StartDate or EndDate
     * @return Date
     */ 
-    private static Date dtStartOrEndOfAddrSeasonal(Address__c addr, boolean fStart) {    
+    private static Date getStartOrEndDateOfAddrSeasonal(Address__c addr, boolean returnStartDate) {    
```

Comment:
I don't think renaming my methods and variables is in the best interest of sharing code, just because you prefer your naming conventions.  I try to always respect the existing naming and syntax conventions of any shared code I am modifying.